config/controller.yaml

```diff
@@ -105,6 +105,8 @@ spec:
           value: config-leader-election
         - name: METRICS_DOMAIN
           value: tekton.dev/pipeline
+        securityContext:
+          runAsUser: 1001
```

Comment:
@skaegi shouldn't `allowPrivilegedEscalation: false` also be there ? :stuck_out_tongue_closed_eyes:

Reply:
In this case it probably does not matter however it's a step in the right direction and for consistency yes I think so. ;)

Reply:
done